git/hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+files=`git diff --cached --name-status`
+re="gen-PKGBUILD.py"
+if [[ $files =~ $re ]]
+then
+  echo "Re-generating PKGBUILD..."
+  python gen-PKGBUILD.py > PKGBUILD
+  git add gen-PKGBUILD.py
+  echo "Re-generating .SRCINFO..."
+  makepkg --printsrcinfo > .SRCINFO
+  git add .SRCINFO
+  git commit --amend -C HEAD --no-verify
```

Comment:
You're right, my bad. Turns out you can remove the commit line entirely and the added files will remain in the commit even though the diff listing in the commit message doesn't show them.